_posts/en/newsletters/2020-07-08-newsletter.md

```diff
@@ -0,0 +1,216 @@
+  LND maintenance release is now available for testing.  It includes
+  several bug fixes, including an important fix related to the creation
+  of backups.
+
+- [LND 0.10.3-beta.rc1][lnd 0.10.3-beta]: this release candidate,
+  separate from the 0.10.2 RC, includes a package refactoring in
+  addition to the bug fixes provided in 0.10.2.  For details, see a
+  mailing list [post][osuntokun rcs] by LND developer Olaoluwa
+  Osuntokun.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19204][] eliminates a source of wasted bandwidth during
+  Initial Block Download (IBD).  When Bitcoin Core is in IBD, it often
+  doesn't have the necessary information to validate currently
+  unconfirmed transactions, so it ignores relayed transaction
+  announcements.  However, those announcements still consume the
+  bandwidth of the receiving IBD node and its sending peers.  This PR
+  uses the [BIP133][] `feefilter` P2P message to tell peers that the
+  node doesn't want to receive announcements of any transactions that
+  pay less than 21 million BTC in fees per 1,000 vbytes, which should
+  prevent it from receiving any legitimate transaction announcements.
+  When the node finishes catching up to the tip of the best block chain,
+  it sends another `feefilter` message with its actual minimum relay
+  feerate so that it can start to receive newly-announced transactions.
+
+- [Bitcoin Core #19215][] creates [PSBTs][topic psbt] containing each
+  previous transaction that created one of the UTXOs being spent in the
+  current transaction---even for segwit v0 UTXOs.  Before this change,
+  previous transactions were only included for legacy (non-segwit)
+  UTXOs.  This change is a response to some hardware wallets now
+  requiring or recommending access to previous transactions for segwit
+  UTXOs in order to mitigate the [fee overpayment attack][].  If segwit
+  v1 ([taproot][topic taproot]) is adopted, transactions spending all
+  taproot inputs should not need this extra data by default.
+
+- [C-Lightning #3775][] new RPCs: reserveinputs unreserveinputs signpsbt sendpsbt FIXME:dongcarl
+
+- [Eclair #1427][] and [#1439][Eclair #1439] add support to Eclair for
+  effective sending of [multipath payments][topic multipath
+  payments]---payments which are split into several parts, with each
+  part routed using a different path.  These PRs split payments into up
+  to six parts by default, starting each part with 0.00015 BTC but
+  increasing the value of each part in turn as much as possible until
+  the full payment amount has been allocated.  This is efficient but,
```

Comment:
nit: Is allocated the right word? Alternatives: s/allocated/sent, s/until
  the full payment amount has been allocated/until the full amount has been paid.